server/functions/iterate.go

```diff
@@ -243,10 +248,11 @@ func iterateTables(ctx *sql.Context, callbacks Callbacks, itemSchema ItemSchema,
 	// Iterate over the sorted table names
 	for _, tableName := range sortedTableNames {
 		table, ok, err := itemSchema.Item.GetTableInsensitive(ctx, tableName)
-		if err != nil {
+		if err != nil && !errors.Is(err, doltdb.ErrTableNotFound) {
 			return err
 		} else if !ok {
-			return sql.ErrTableNotFound.New(tableName)
+			// We receive these names from the database, so these must be the names of root objects
```

Comment:
It would be better if we only passed actual table names to this function

Reply:
We need to eventually change how this is done anyway